internal/cluster/cluster.go

```diff
@@ -562,3 +573,28 @@ func (m *Manager) IsReadyForInstallation(c *common.Cluster) (bool, string) {
 	}
 	return true, ""
 }
+
+func (m *Manager) DeleteClusterLogs(ctx context.Context, c *common.Cluster, objectHandler s3wrapper.API) error {
+	log := logutil.FromContext(ctx, m.log)
+	files, err := objectHandler.ListObjectsByPrefix(ctx, fmt.Sprintf("%s/logs/", c.ID))
+	if err != nil {
+		return common.NewApiError(http.StatusNotFound, err)
+	}
+	for _, file := range files {
+		log.Debugf("Deleting cluster %s S3 log file: %s", c.ID.String(), file)
+		if err := objectHandler.DeleteObject(ctx, file); err != nil {
```

Comment:
Maybe this deletion can be done with ttl and then we can call this function on cluster deletion (and not a week after)?

Same for `DeleteClusterFiles` func.

Reply:
What do you mean "done with ttl"?

Reply:
We thought maybe on cluster deletion to define a ttl for the objects in s3. But it's not relevant, won't effect anything as I see it.